
int32 iteration_id

---
Path[]  paths
bool is_exploration_finish

---

bool is_get_path